addons/portal/wizard/mail_invite_share.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, fields, models, _
+
+
+class MailInviteShare(models.TransientModel):
+    _name = 'mail.invite.share'
+
+    @api.model
+    def default_get(self, fields):
+        result = super(MailInviteShare, self).default_get(fields)
+        result['model'] = self._context.get('active_model')
+        result['res_id'] = self._context.get('active_id')
+        return result
```

Comment:
in this method, with active_model and active_id, browse the object, and call 'get_share_url' to get the url access, and put it in 'share_link' field.